iotileship/test/test_jsonstep.py

```diff
@@ -0,0 +1,79 @@
+import json
+import pytest
+from iotile.ship.actions.ModifyJsonStep import ModifyJsonStep
+
+class mock_resource:
+    def __init__(self, root):
+        self.root = root
+
```

Comment:
I'm not sure how strict we are on test code being pep8, but top-level functions/classes should have 2 blank lines between them (vs. one line for class methods)